package.json

```diff
@@ -12,15 +12,15 @@
     "prettier": "2.8.8"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap --no-ci --since master",
-    "bootstrap:ci": "lerna bootstrap --ci --concurrency=1 --since master",
+    "bootstrap": "lerna bootstrap --no-ci --since master --include-dependencies",
```

Comment:
@pkeavenycontentful and I identified this flag as being necessary in his serverless upgrade work

It ended up being necessary in this as well, so I've added it here.

https://lerna.js.org/docs/api-reference/commands#--include-dependencies